packages/wmr/test/production.test.js

```diff
@@ -889,6 +889,40 @@ describe('production', () => {
 			const index = await fs.readFile(indexHtml, 'utf8');
 			expect(index).toMatch(/# hello world/);
 		});
+
+		// Whether or not to run the test to fetch external resources depends on whether or not a Node.js version 18 later
+		const isNodeVersionOver18 = Number(process.version.split('.')[0].slice(1)) >= 18;
+		// Even if you are using Node.js v18 or higher, you will get the error `globalThis.fetch is undefined` here, so get the version from process.version
+		// if (globalThis.fetch === undefined)
+		if (!isNodeVersionOver18) {
+			it.skip('skip should not support fetching resources from external link prerender when Node.js v18 later', () => {});
+			it('should not support fetching resources from external link prerender when Node.js befor v17', async () => {
+				await loadFixture('prerender-external-resource-fetch', env);
+				instance = await runWmr(env.tmp.path, 'build', '--prerender');
+				const code = await instance.done;
+
+				await withLog(instance.output, async () => {
+					expect(code).toBe(1);
+					expect(instance.output.join('\n')).toMatch(
+						/fetch is not defined in Node.js version under 17, please upgrade to Node.js version 18 later/
+					);
+				});
+			});
+		} else {
+			it.skip('skip should not support fetching resources from external link prerender when Node.js befor v17', () => {});
+			it('should not support fetching resources from external link prerender when Node.js v18 later', async () => {
```

Comment:
This seems odd? On Node 18 it shouldn't support fetching external resources? Probably just a mistake in the test name?

Reply:
Oops, that was my mistake! Thanks!